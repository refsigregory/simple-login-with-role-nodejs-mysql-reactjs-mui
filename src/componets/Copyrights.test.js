import React from 'react';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';

import { Copyright } from './Copyrights';

it('renders correctly', () => {
  const tree = renderer
    .create(<Copyright />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

test('renders copy rights', () => {
  const { getByText } = render(
    <Copyright />
  );

  expect(getByText(/Copyright Â©/i)).toBeInTheDocument();
});

test('renders My Website', () => {
  const { getByText } = render(
      <Copyright />
  );

  expect(getByText(/My Website/i)).toBeInTheDocument();
});