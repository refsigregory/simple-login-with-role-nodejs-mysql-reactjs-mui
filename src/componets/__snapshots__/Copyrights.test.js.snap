// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<p
  className="MuiTypography-root MuiTypography-body2 MuiTypography-alignCenter css-1vl95ex-MuiTypography-root"
>
  Copyright Â© 
  <a
    className="MuiTypography-root MuiTypography-inherit MuiLink-root MuiLink-underlineAlways css-1cghntr-MuiTypography-root-MuiLink-root"
    href="https://ref.si/"
    onBlur={[Function]}
    onFocus={[Function]}
  >
    My Website
  </a>
   
  2022
  .
</p>
`;
